Tangem/Modules/TokenDetails/TokenDetailsView.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  TokenDetailsView.swift
+//  Tangem
+//
+//  Created by Andrew Son on 09/06/23.
+//  Copyright © 2023 Tangem AG. All rights reserved.
+//
+
+import SwiftUI
+
+struct TokenDetailsView: View {
+    @ObservedObject var viewModel: TokenDetailsViewModel
+
+    var body: some View {
+        RefreshableScrollView(onRefresh: viewModel.onRefresh) {
+            VStack(spacing: 14) {
+                BalanceWithButtonsView(viewModel: viewModel.balanceWithButtonsModel)
+            }
+        }
+        .padding(.horizontal, 16)
+        .edgesIgnoringSafeArea(.bottom)
+        .navigationBarTitle("", displayMode: .inline)
+        .navigationBarItems(trailing: navbarTrailingButton)
+        .background(Colors.Background.secondary.edgesIgnoringSafeArea(.all))
+        .ignoresSafeArea(.keyboard)
+        .onAppear(perform: viewModel.onAppear)
+        .alert(item: $viewModel.alert) { $0.alert }
+    }
+
+    @ViewBuilder
+    private var navbarTrailingButton: some View {
+        Menu {
+            if #available(iOS 15.0, *) {
+                Button(Localization.tokenDetailsHideToken, role: .destructive, action: viewModel.hideTokenButtonAction)
+            } else {
+                Button(Localization.tokenDetailsHideToken, action: viewModel.hideTokenButtonAction)
+            }
```

Comment:
Так а может одинаковую кнопку сделать, что то типо

```suggestion
           Button(action: viewModel.hideTokenButtonAction) {
                Text(Localization.tokenDetailsHideToken)
                     .foregroundColor(.red) 
           }
```

Reply:
они в менюшке почему-то не перекрашиваются и единственный вариант Кастомизацию появился в 15 оси, поэтому так пришлось делать